notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1124 @@
+{
```

Comment:
<p>Is there any significant difference between this dataset and the dataset used for classification with AutoML (https://pantheon.corp.google.com/storage/browser/_details/cloud-ml-tables-data/bank-marketing.csv)? Is there any motivation for choosing one over another?</p><p> </p><p>It looks like we take a dataset called train.csv and then split it into test/train/evaluation sets. But the kaggle source provides a test.csv in addition to a train.csv. If we want to do predictions / explanations, will we use the test split or test.csv?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
I'm not familiar with the bank-marketing dataset, but I used Safe Driver because it's the dataset that the automl_tabular_on_vertex_pipelines.ipynb notebook uses. 

Yes, we take the train.csv from the kaggle source and split it into test/train/evaluation sets. The pipeline doesn't support explanations, but model evaluation, for example, will run on the test set we created. If a customer wanted to deploy the model or do batch prediction after the pipeline completes, they could use test.csv.